### DISCUSSION TITLE
Release Year under Album

### DISCUSSION THREAD

**Discussion by DeN-AlB** [ciderapp/Cider#1396] [Feature Request]
### Describe the Bug

When looking through the releases of an artist, there is no release year shown below the title, as it is on Apple Music Web.

Apple Music Web in browser:
![image](https://user-images.githubusercontent.com/55697152/186417942-6411f970-5346-4135-b881-771530a84f96.png)

Cider:
![image](https://user-images.githubusercontent.com/55697152/186418006-efbabc26-30b5-4581-94ca-1f47133a1bd4.png)

I tried to use different styles, but the release year isn't shown also.

### Steps to Reproduce

See description of the bug.

### Environment Information

- OS: Windows 11 Pro 21H2 (22000.856)
- App Version: 1.5.5 beta 114
- Browser: Vivaldi 5.4.2753.37 (Stable channel) (64-Bit)

### Code of Conduct

- [X] I agree to follow this Cider's [Code of Conduct](https://github.com/ciderapp/Cider/blob/main/CODE_OF_CONDUCT.md)

### Disclaimer

- [X] I acknowledge the [Support Disclaimer](https://docs.cider.sh/support/disclaimer)

**Comment by shiraari** [comment]
Cider isn't 1:1 Apple Music Web, it looks similar, but it's not the same thing, I wouldn't call it a "bug"

**Comment by DeN-AlB** [comment]
Good to know. Maybe I had to mark it as a feature request then ...
